node/tests/SharedTests.ts

```diff
@@ -4424,6 +4424,16 @@ export function runBaseTests<Context>(config: {
                 // default geoadd
                 expect(await client.geoadd(key1, membersToCoordinates)).toBe(2);
 
+                const geopos = await client.geopos(key1, [
```

Comment:
should we add any of the following:
- passing a non-existing key
- passing a non-existing member
- pass empty list of members

Reply:
Added